### ISSUE TITLE
"Possibly searching too often, take a rest."

### ISSUE THREAD

**Issue by pogarek** [PokemonGoF/PokemonGo-Bot#5821]
HI
Since two days my bot is getting blocked by captcha..
The bot console shows a message "Possibly searching too often, take a rest." . When I login the account to the real game, I get captcha. After fixing captcha and starting bot again, after 10-15 minutes bot is blocked again. 
Config, IPs hasn't been changed in last 5-7 days. I don't see anything in this area, which can cause the issue. I've also tried to start the bot in another network - the same block happened after similar time.

Any thoughts how to find the reason /fix it ?

**Comment by jkailasam** [comment]
I have the same problem for the last 2 days.

**Comment by enginebai** [comment]
Watch out for being banned......

**Comment by MerlionRock** [comment]
Did you run your bot 24hr or did you enable the random pause & sleep?

**Comment by pogarek** [comment]
Yes, the bot is going to sleep twice per 24 hours day. For approximately 3 and 6 hours of sleep.

**Comment by Cptspal** [comment]
which speed ?
I ran a bot from several days without a pause, just the limit of catched pokemon and pokestop, and no captcha. But it's walking quite slowly :
    "walk_max": 3.16,
    "walk_min": 1.16,

**Comment by pogarek** [comment]
"walk_max": 4.16,
    "walk_min": 2.16,

Basically defaults from config.json.example

**Comment by MerlionRock** [comment]
Me have not seen the captcha too.

But, as @cptspal has mentioned, I've decreased the speed and use polyline Walker.

Also in case I meet captcha, I've also written a script to get the URL of the captcha and wait for token input. But because I have not yet meet the challenge, therefore I'm not sure if I have coded it correctly.

**Comment by Cptspal** [comment]
I've limited the speed of bots because of buddy's candy, the default speed is too much to work properly specially when you don't have too much pokemon to catch or pokestop to spin.

**Comment by jkailasam** [comment]
This is happening because of the Captcha. There is already a fix in the Pokemongo Map project for the Captcha. Wondering how to implement it in Pokemongo Bot. 

https://github.com/PokemonGoMap/PokemonGo-Map/pull/1526

**Comment by pogarek** [comment]
I have changed the walker settings from 2.16/4.16 to 1.16/3.16 . Bot displayed the same message (which means captcha) after 45 minute of work and the bot wasn't running in last two days..

**Comment by goga22** [comment]
The bot is running 0.45 API and now https://pgorelease.nianticlabs.com/plfe/version says that the latest is 0.47. Maybe that's the reason for captcha.

**Comment by ikariya69** [comment]
same problem here, the bot does not see any pokestops nor pokemons since yesterday

**Comment by Aelfwin1996** [comment]
anyone found anything that helps the bot to run longer (without hitting Captcha) than 40 mins on the lower speed these days (aside from bossland stuff)?

**Comment by MerlionRock** [comment]
@aelfwin, Niantic detects not just behavior, but also how you login. By not using latest API, that account is already flagged and it will hit captcha no matter what setting you use.
